src/DataLayer/converters/MeasureConverter.ts

```diff
@@ -147,11 +148,15 @@ function convertPreviousPeriodMeasureDefinition(definition: IPreviousPeriodMeasu
     };
 }
 
-function convertMeasureDefinitionFormat(definition: IMeasureDefinitionType): string | null {
-    if (VisualizationObject.isMeasureDefinition(definition)) {
-        return convertSimpleMeasureDefinitionFormat(definition);
-    }
-    return null;
+function getFormat(measure: IMeasure): string | undefined {
+    const { measure: { definition } } = measure;
+    const measureFormat = get(measure.measure, 'format');
+
+    const predefinedFormat = VisualizationObject.isMeasureDefinition(definition)
+        ? convertSimpleMeasureDefinitionFormat(definition)
```

Comment:
hmm probably one extra change could you please change the function name to something else?

for example `getPredefinedFormat` ?